### DISCUSSION TITLE
How to gather logs from worker POD in Kubernetes - Celery Executor (Bug?)

### DISCUSSION THREAD

**Discussion by MattiaGallegati** [apache/airflow#24579] [Q&A]
Hello everyone,
As part of my "fresh" Airflow installation I've been able to deploy it on Kubernetes using the provided Helm Chart.
My setup uses Celery Executor with one worker on the cluster and one outside the cluster.
I've been able to gather logs from the remote worker (the one outside the cluster) manually configuring a Kubernetes Service, yet I can't gather logs from the "embedded" one on the cluster.
The message is:
"Failed to fetch log file from worker. [Errno -2] Name or service not known".
It seems like there is no "service" in order to correctly identify the worker address. 
I've checked my kube resources and I can see a Service for that worker actually exists.
My configurations has persistence to False.
Any advice on how to achieve that?

**Comment by hugochinchilla** [comment]
I'm experiencing exactly the same issue using airflow 2.3.2 with CeleryExecutor, also running form the helm chart.

**Reply by potiuk** [reply]
If you need any help - you need to provide more information. tehre is no way anyone can help without seeing either logs ar knowing configuraiton for aither of you. Some deeper investigation is needed and more evidence, if you want somoene to help you with your problems.

**Reply by hugochinchilla** [reply]
The problem was solved by setting `core.hostname_callable` to `airflow.utils.net.get_host_ip_address`

**Reply by MattiaGallegati** [reply]
@hugochinchilla thank you!
